api/src/main/resources/workbench.yaml

```diff
@@ -724,7 +728,54 @@ paths:
           type: integer
           format: int64
           required: true
-          description: specifies version of cdr
+          description: specifies which cdr version
+      responses:
+        "200":
+          description: A cohortReviewId, creationTime and cohortCount
+          schema:
+            $ref: "#/definitions/CohortReviewInfo"
+
+  /api/v1/cohortreview/create/{cohortReviewId}:
```

Comment:
Similar to above: the REST path could include the cohort ID. If the ID is globally unique it technically doesn't have to.